### DISCUSSION TITLE
请问V5是否还支持像geosite:apple这样的数据格式？

### DISCUSSION THREAD

**Discussion by Benjamin1919** [IrineSistiana/mosdns#808] [Q&A]
v2ray/xray等代理软件可以用geosite:apple、geosite:google、geosite:cn等数据标签来指代一系列域名，
但是我看了好几个V5的配置示例，大家都是在数据插件里使用自建的txt文件来储存域名列表，

请问外部文件是否支持数据标签形式的引用？比如：
`./domain.txt:apple`

如果不能，有没有较快的方法把geosite:apple导出为txt文件，这样geosite.dat源文件更新后，存储域名的txt也能跟着快速更新？

**Comment by Sereinfy** [comment]
写个脚本 定时更新。就行
curl url > apple.txt 然后relord mosdns
参考easymosdns updata文件
或者用作者的另一个工具，解包geosite 两行命令的事。
https://github.com/urlesistiana/v2dat
